### ISSUE TITLE
New tab focuses the awesome bar but it's covered if you just start typing

### ISSUE THREAD

**Issue by Verdi** [bwinton/VerticalTabs#89]
STR:
1. Click new tab and don't move your pointer
2. Starting typing your search or url
3. Tab Center UI covers the beginning of the awesome bar and hides what you're typing
![tabcenter](https://cloud.githubusercontent.com/assets/107992/14924034/182bcba4-0e06-11e6-9b68-dbd91d57f1f9.gif)

**Comment by liljaylj** [comment]
Please reopen this issue. Because it's not resolved. Tab Center UI still covers the beginning of the awesome bar and hides what you're typing. And this is not duplicate of #102

**Comment by Keith94** [comment]
You're mistaken because the issue is still open, it just hasn't been fixed yet. Fixing bug #142 would address this, though.

**Comment by chuckharmston** [comment]
> Please reopen this issue. Because it's not resolved

It's actually still open; a few pull requests (#358, #102) that have mentioned this bug have been closed.
